pkg/aws/services/vpclattice.go

```diff
@@ -13,6 +14,12 @@ import (
 
 //go:generate mockgen -destination vpclattice_mocks.go -package services github.com/aws/aws-application-networking-k8s/pkg/aws/services Lattice
 
+type Tags = map[string]*string
+
+type ServiceNetworkInfo struct {
+	SvcNetwork *vpclattice.ServiceNetworkSummary
```

Comment:
why reference? ServiceNetworkInfo returned below as reference too, I would assume that when we return ServiceNetworkInfo content should be valid, and there is no need for extra nullity check.

Reply:
Good point. Updated.